### PR TITLE
Added example for hashing and signature verification

### PR DESCRIPTION
**Pull request by laishawadhwa**
Closes Issue #37 
- Added following under examples
    - Hashing
    - Signature verification

### CODE REVIEW

File: examples/verify_signature/src/main.sw [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+use std::ecr::{ec_recover_address, EcRecoverError};
+use std::bytes::Bytes;
+use std::b512::B512;
+use std::constants::ZERO_B256;
+
+abi VerifySignature {
+    fn get_message_hash(_to: b256, _amount: u64, _message: str, _nonce: u64) -> b256;
+    fn get_eth_signed_message_hash(_message_hash: b256) -> b256;
+    fn recover_signer(_eth_signed_message_hash: b256, _signature: B512) -> b256;
+    // NEEDS TO BE FIXED
```
Comment (calldelegation) [inline examples/verify_signature/src/main.sw]: What needs to be fixed?

**Review by calldelegation — CHANGES_REQUESTED** [review]
1. hashing.md missing in SUMMARY.md
2. Lint CI is not updated
3. Add verify_signature and hashing to actions.yml so it checks for compilation
4. Add Keccak to words for spell check

File: examples/verify_signature/src/main.sw (hunk 1 above)
Comment (laishawadhwa) [reply examples/verify_signature/src/main.sw]: There was a bug when I first pushed it. Will remove the comment

**Comment by laishawadhwa** [comment]
@calldelegation : Added the requested changes

**Pull request merged**